internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -118,7 +123,7 @@ func (v *vmConsoleProxy) Cleanup(request *common.Request) ([]common.CleanupResul
 
 func reconcileServiceAccountsFuncs(serviceAccount core.ServiceAccount) common.ReconcileFunc {
 	return func(request *common.Request) (common.ReconcileResult, error) {
-		serviceAccount.Namespace = kubeVirtNamespace
+		serviceAccount.Namespace = getVmConsoleProxyNamespace(request)
 		return common.CreateOrUpdate(request).
 			ClusterResource(&serviceAccount).
 			WithAppLabels(operandName, operandComponent).
```

Comment:
Can you add `UpdateFunc()` here, and copy all the fields from the `newRes` to `foundRes`? Except `TypeMeta` and `ObjectMeta`. It is needed to correctly reconcile the reosurce.

Reply:
Yep, done.